libr/core/disasm.c

```diff
@@ -1009,6 +1009,9 @@ static void ds_build_op_str(RDisasmState *ds, bool print_color) {
 	core->parser->relsub = r_config_get_i (core->config, "asm.relsub");
 	core->parser->regsub = r_config_get_i (core->config, "asm.regsub");
 	core->parser->relsub_addr = 0;
+	if (core->parser->relsub && ds->analop.type == R_ANAL_OP_TYPE_LEA && ds->analop.ptr != UT64_MAX) {
```

Comment:
why restrict on lea?

Reply:
Umm the only test case i have uses lea -- test cases can be generated for other instructions though